src/lib/draw-and-pick.js

```diff
@@ -95,16 +96,18 @@ export function pickLayers(gl, {
 
   } else {
     // For all other events, run picking process normally.
-    const pickInfo = pickFromBuffer(gl, {
+    const pickResult = pickFromBuffer(gl, {
       layers,
       pickingFBO,
       deviceX,
-      deviceY
+      deviceY,
+      deviceRadius
     });
-    pickedColor = pickInfo.pickedColor;
-    pickedLayer = pickInfo.pickedLayer;
-    pickedObjectIndex = pickInfo.pickedObjectIndex;
-    affectedLayers = pickInfo.affectedLayers;
+
+    pickedColor = pickResult.color;
+    pickedLayer = pickResult.layer;
+    pickedObjectIndex = pickResult.objectIndex;
+    affectedLayers = pickedLayer ? [pickedLayer] : [];
```

Comment:
Yep, I already moved this out in the drag events PR because we don't always pick -- drag events that indicate the continuation of a drag event that started on a specific element don't need to re-pick. In that case, the user is assumed to be dragging an element around and so the element picked on `dragstart` is returned without picking.